extensions/amp-analytics/0.1/visibility-impl.js

```diff
@@ -460,35 +467,24 @@ export class Visibility {
 
     // Remove target that have no listeners.
     if (listeners.length == 0) {
-      this.intersectionObserver_.unobserve(change.target);
+      this.intersectionObserver_.unobserve(target);
```

Comment:
To double-check: `observe` is called for each trigger and `onubserve` is called only once after all triggers are done. Correct?